openssl/src/symm.rs

```diff
@@ -1248,6 +1248,7 @@ mod tests {
     }
 
     #[test]
+    #[cfg(not(ossl300))]
```

Comment:
It seems surprising to me that this PR needs to add this, why is it not affecting everything else?